android/app/src/main/java/com/mobile/MainApplication.java

```diff
@@ -21,17 +19,6 @@ public boolean getUseDeveloperSupport() {
           return BuildConfig.DEBUG;
         }
 
-        @Override
-        protected List<ReactPackage> getPackages() {
-          @SuppressWarnings("UnnecessaryLocalVariable")
-          List<ReactPackage> packages = new PackageList(this).getPackages();
-
-          // Packages that need to be manually linked
-          packages.add(new WatermelonDBPackage());
-
-          return packages;
-        }
```

Comment:
Just need to remove Line#30 and keep this functions. This is what will add packages that RN brings (from what is understand from this function because without this android build fails)